episerver/Quicksilver
# Origin/feature/traf 19 setup

Traffee Setup Script

JIRA issue: https://weareinspire.atlassian.net/browse/TRAF-19


## Prerequisites/How to test
1. Edit application configuration as needed:
![image](https://user-images.githubusercontent.com/4353148/63008525-6129d400-beb5-11e9-88e0-dd675d3f5b8d.png)

2. Publish **Setup** project
![image](https://user-images.githubusercontent.com/4353148/63008833-f1681900-beb5-11e9-84bd-346aaf314fa4.png)

2. Make sure you have Admin privileges when you run the Setup.exe.
![image](https://user-images.githubusercontent.com/4353148/63008972-32f8c400-beb6-11e9-9810-377b97910b29.png)



## Test ideas
- [ ] Check web application in IIS Manager if added:
![image](https://user-images.githubusercontent.com/4353148/63009149-85d27b80-beb6-11e9-8482-2417adfe8a39.png)

- [ ] Check the following databases if added:
![image](https://user-images.githubusercontent.com/4353148/63009267-b74b4700-beb6-11e9-9be1-1ed7be4d3fd2.png)

- [ ] Check if Node.js was installed
![image](https://user-images.githubusercontent.com/4353148/63009312-d4801580-beb6-11e9-8c7a-aa2cfd7d94ab.png)


## Known issues/things not included
- [[...]]

## Checklist
- [ ] No code checked in that is commented out
- [ ] No hard coded dev values (emails, postal codes, names, etc)
- [ ] Code handles errors gracefully and the user is notified when such errors occur


## Screen dumps
